### PR TITLE
Deployment gradle maven

### PR DESCRIPTION
**Pull request by andreyaksenov**
Created tutorials showing how to use the Gradle Application plugin and Maven Assembly plugin to package and run a Ktor application.

### CODE REVIEW

File: topics/gradle-application-plugin.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+<p>
+<control>Sample project</control>: <a href="https://github.com/ktorio/ktor-gradle-sample/tree/main">ktor-gradle-sample</a>
+</p>
+</microformat>
+
+The Gradle [Application plugin](https://docs.gradle.org/current/userguide/application_plugin.html) provides the ability to package applications, including code dependencies and generated start scripts. In this topic, we'll show you how to package and run a Ktor application created in the [](Gradle.xml) topic.
+
+## Prerequisites {id="prerequisites"}
+Before starting this tutorial, do the following:
+* [Install IntelliJ IDEA](https://www.jetbrains.com/help/idea/installation-guide.html)
```
Comment (Stexxe) [inline topics/gradle-application-plugin.md:13]: Is it a necessary step?

File: topics/gradle-application-plugin.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Clone the [ktor-gradle-sample](https://github.com/ktorio/ktor-gradle-sample) repository.
+
+
+## Apply the Application plugin and configure the main class {id="apply-plugin"}
+To package an application, you need to apply the Application plugin first:
+1. Open the [ktor-gradle-sample](https://github.com/ktorio/ktor-gradle-sample) project in IntelliJ IDEA.
```
Comment (Stexxe) [inline topics/gradle-application-plugin.md:19]: Is it necessary to open a project in the IDE to accomplish packaging?

File: topics/gradle-application-plugin.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+1. Make sure that the `build.gradle` file contains the following code:
+   ```groovy
+   ```
+   {src="https://raw.githubusercontent.com/ktorio/ktor-gradle-sample/main/build.gradle" lines="1,3-4,8,9-11"}
+    * The `id 'application'` line applies the Application plugin.
```
Comment (Stexxe) [inline topics/gradle-application-plugin.md:24]: I am just curious, could we not duplicate content by marking some lines with numbers and refer them later?

File: topics/gradle-application-plugin.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+   {src="https://raw.githubusercontent.com/ktorio/ktor-gradle-sample/main/build.gradle" lines="1,3-4,8,9-11"}
+    * The `id 'application'` line applies the Application plugin. 
+    * The `mainClass` property is used to configure the main class of the application.
+
+
+## Package the application {id="package"}
+The Application plugin provides various ways for packaging the application, for example, the `installDist` task installs the application with all runtime dependencies and start scripts. To create full distribution archives, you can use the `distZip` and `distTar` tasks.
+
+In this topic, we'll use `installDist`:
+1. Open the [terminal](https://www.jetbrains.com/help/idea/terminal-emulator.html).
```
Comment (Stexxe) [inline topics/gradle-application-plugin.md:32]: Again, this could be done with any terminal emulator

File: topics/maven-assembly-plugin.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+[//]: # (title: Maven Assembly plugin)
+
+<microformat>
+<p>
+<control>Initial project</control>: <a href="https://github.com/ktorio/ktor-maven-sample/">ktor-maven-sample</a>
```
Comment (Stexxe) [inline topics/maven-assembly-plugin.md:5]: Why don't we have such a thing for the Gradle Application plugin﻿ topic?

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: topics/gradle-application-plugin.md (hunk 1 above)
Comment (andreyaksenov) [reply topics/gradle-application-plugin.md:13]: ok, I'll remove this requirement from tutorials

File: topics/maven-assembly-plugin.md (hunk 5 above)
Comment (andreyaksenov) [reply topics/maven-assembly-plugin.md:5]: because for Gradle we take a ready project and don't change anything there

**Review by Stexxe — APPROVED** [review]

**Pull request merged**